resources/data/llama3-tokenizer/special_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "enclose_eos_bos": true
```

Comment:
This generally won't work for the case where a Llama model's tokenizer files are downloaded.